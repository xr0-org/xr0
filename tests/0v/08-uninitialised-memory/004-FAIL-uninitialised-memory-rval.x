int
func(int i) ~ [
	return i;
]{
	return i;
}

int
main()
{
	int i;
	func(i);
}
