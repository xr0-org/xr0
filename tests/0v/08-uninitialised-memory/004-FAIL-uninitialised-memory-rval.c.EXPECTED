08-uninitialised-memory/004-FAIL-uninitialised-memory-rval.x:20:9: undefined memory access: i has no value (main)
